spec/cell_spec.rb

```diff
@@ -81,3 +81,59 @@
     end
   end
 end
+
+RSpec.describe Cell do
+  context 'Render cell' do
+    it 'renders as . by default' do
+      cell_1 = Cell.new('B4')
+
+      expect(cell_1.render).to eq('.')
+    end
+
+    it 'renders as M if fired upon and no ship' do
+      cell_1 = Cell.new('B4')
+
+      cell_1.fire_upon
+
+      expect(cell_1.render).to eq('M')
+    end
+
+    it 'renders as . if not fired upon' do
+      cell_2 = Cell.new('C3')
+      cruiser = Ship.new('Cruiser', 3)
+      cell_2.place_ship(cruiser)
+
+      expect(cell_2.render).to eq('.')
+    end
+
+    it 'renders as S if not fired upon and show_ship is true' do
+      cell_2 = Cell.new('C3')
+      cruiser = Ship.new('Cruiser', 3)
+      cell_2.place_ship(cruiser)
+
+      expect(cell_2.render(true)).to eq('S')
+    end
+
+    it 'renders as H if ship is hit' do
+      cell_2 = Cell.new('C3')
+      cruiser = Ship.new('Cruiser', 3)
+      cell_2.place_ship(cruiser)
+      cell_2.fire_upon
+      cruiser.sunk?
```

Comment:
We should change `cruiser.sunk? ` to a test here to show that it returns false, and in the next test to show that it returns true. If the interaction pattern shows the return, I think we should replicate it in the tests.

Reply:
Fixed, I think

Reply:
You can remove `cruiser.sunk?` in lines 122 and 135 because they aren't doing anything.

Reply:
Should be all clear now